lib/features/domain/repositories/local_repository.dart

```diff
@@ -1 +1,7 @@
 // TODO: Add a local repository containing use case functions' definition
+import '../entities/question_paper_entity.dart';
+
+abstract class LocalRepository{
+  Future<void> searchQuestionPaper(QuestionPaperEntity questionPaperEntity);
```

Comment:
`fetchQuestionPaper` would be a better name for the function. We're not providing a string search right?